runtime-light/coroutine/awaitable.h

```diff
@@ -199,3 +200,41 @@ class wait_fork_t {
     }
   }
 };
+
+// ================================================================================================
+
+class wait_queue_next_t {
```

Comment:
Will it work with job workers? Job workers are also represented as futures, so we can wait on them as well.

Reply:
fix it, check `wait_queue_t`